features/facebookImport/src/components/importExplanationExpandable/importExplanationExpandable.jsx

```diff
@@ -48,6 +48,18 @@ const ImportExplanationExpandable = ({
         setStartRequest(true);
     };
 
+    const exampleDataUrls = {
+        en: "https://polypoly.coop/en-de/polypod/example-facebook-data",
+        de: "https://polypoly.coop/de-de/polypod/example-facebook-data",
+    };
+
+    const handleExampleDataRequest = () => {
+        onUpdateImportStatus(importSteps.download);
+        const url = exampleDataUrls[i18n.language] || exampleDataUrls.en;
+        window.pod.polyNav.openUrl(url);
+        setStartRequest(true);
```

Comment:
I suppose this shouldn't move people to the second step, but directly the third, right?

Reply:
Yes, updated to redirect to the import step.

Reply:
IIRC, weren't we using pod as an argument instead of using directly `window.pod`?

Reply:
For now we only made that refactoring in importers and mini-stories.

Reply:
> IIRC, weren't we using pod as an argument instead of using directly `window.pod`?

Normally yes, but it doesn't matter a whole lot - what really matters is that this code runs after `window.pod` is available (which we ensure already during feature initialisation).